standard_test.go

```diff
@@ -87,7 +88,7 @@ func TestPlaceSnakes(t *testing.T) {
 		},
 		{
 			&BoardState{
-				Width:  1,
+				Width:  2,
```

Comment:
Same with these test cases, can you leave the original tests intact, possibly changing the expected results (instead of the inputs) and then add new ones?

Reply:
done